src/main/java/cn/edu/tsinghua/tsfile/timeseries/read/LocalFileInput.java

```diff
@@ -30,7 +32,17 @@ public int read() throws IOException {
     }
 
     public int read(byte[] b, int off, int len) throws IOException {
-        return raf.read(b, off, len);
+    		int end=len+off;
+    		int get=1;
+    		int total=0;
+    		for(int i=off;i<end; i+=get) {
+    			get=raf.read(b, i, end-i);
+    			if(get>0)
+    				total+=get;
+    			else
+    				break;
+    		}
+        return total;
```

Comment:
(1) The code should be formatting.
(2) And if you read the ```byte[] b``` only a byte one time,  would the efficiency be lower than ```raf.read(b, off, len)```?